### ISSUE TITLE
TestFermiLATBasicImageEstimator.test_run quantity issue

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammapy#1117]
@adonath - Locally I see this Quantity-related AttributeError from `TestFermiLATBasicImageEstimator.test_run quantity`:
https://gist.github.com/cdeil/cef2334aaea5b75a601264b8c57b3ea6

I think this is in code that you wrote. Can you have a look, please?
If you can't reproduce or know what's going on, please comment and assign back to me and I'll investigate / fix.

Possibly this is only visible with very recent Astropy, and that's why it doesn't appear on travis-ci:
```
Running tests with gammapy version 0.7.dev4955.
Numpy: 1.13.1
astropy: 3.0.dev19816
```

**Comment by adonath** [comment]
The issue is the one [explained here](https://github.com/astropy/astroplan/pull/307#issuecomment-314710914). Fixed directly in master with 5eee87f.

**Comment by cdeil** [comment]
@adonath - Thanks!
